### PR TITLE
PMM-13217 Show mongos version and rename the dashboard

### PR DESCRIPTION
**Pull request by YashSartanpara1**
[PMM-13227](https://perconadev.atlassian.net/browse/PMM-13227)
[PMM-13217](https://perconadev.atlassian.net/browse/PMM-13217)

[PMM-13227]: https://perconadev.atlassian.net/browse/PMM-13227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PMM-13217]: https://perconadev.atlassian.net/browse/PMM-13217?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: dashboards/Experimental/MongoDB_Cluster_Summary_(new).json [hunk 1]
```diff
@@ -3088,8 +2906,8 @@
         "allFormat": "glob",
         "current": {
           "selected": false,
-          "text": "mycluster2",
-          "value": "mycluster2"
+          "text": "test",
```
Comment (matejkubinec) [inline dashboards/Experimental/MongoDB_Cluster_Summary_(new).json:2909]: Is this correct?

File: dashboards/Experimental/MongoDB_Cluster_Summary_(new).json [hunk 2]
```diff
@@ -3400,8 +3040,8 @@
       "30d"
     ]
   },
-  "timezone": "",
-  "title": "MongoDB Cluster Summary (New)",
+  "timezone": "browser",
```
Comment (matejkubinec) [inline dashboards/Experimental/MongoDB_Cluster_Summary_(new).json:3043]: Should the timezone be set?

File: dashboards/Experimental/MongoDB_Cluster_Summary_(new).json (hunk 1 above)
Comment (YashSartanpara1) [reply dashboards/Experimental/MongoDB_Cluster_Summary_(new).json:2909]: Hi, this is dynamic and changes per instance.

File: dashboards/Experimental/MongoDB_Cluster_Summary_(new).json (hunk 2 above)
Comment (YashSartanpara1) [reply dashboards/Experimental/MongoDB_Cluster_Summary_(new).json:3043]: Yes. as per the dashboard guidelines "local browser timezone for dashboards"

**Review by matejkubinec — APPROVED** [review]

**Pull request merged**